tests/tests-common/src/main/java/com/hazelcast/simulator/tests/AbstractTest.java

```diff
@@ -25,6 +29,11 @@
  */
 public abstract class AbstractTest {
 
-    protected final ILogger logger = Logger.getLogger(getClass());
+    public final ILogger logger = Logger.getLogger(getClass());
 
+    @InjectHazelcastInstance
+    public HazelcastInstance targetInstance;
```

Comment:
Can you make this fields at least `protected`? Maybe this will be enough for SonarQube.